lib/talon/concern.ex

```diff
@@ -306,6 +306,7 @@ defmodule Talon.Concern do
   Note: This function is overridable
   """
   @spec nav_action_links(atom, atom, Struct.t | Module.t) :: List.t
+  def nav_action_links(_concern, _action, _resource = nil), do: []
```

Comment:
Oops, I meant `nil = _resource`, and I used it over `nil` alone to explicitly document this handled the scenario of a nil resource. Is there a better way here?